.tekton/run-acceptance-test.yaml

```diff
@@ -0,0 +1,86 @@
+---
+apiVersion: tekton.dev/v1beta1
+kind: PipelineRun
+metadata:
+  name: acceptance-test
```

Comment:
please rename to `acceptance-tests-pr`, even add PR number to the name if possible

Reply:
Changed the name but to get the pr number I need to set the webhook in this repo and I don't have the permission to add

Reply:
I don't understand, can't you just use `{{pull_request_number}}`?

Reply:
Yes worked , Added